### PR TITLE
Fix partially supported callables deprecation

### PR DESCRIPTION
**Pull request by jrmajor**
https://wiki.php.net/rfc/deprecate_partially_supported_callables

### CODE REVIEW

File: src/PolyfillTrait6.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ trait PolyfillTrait6
 {
     public function expectException($exception)
     {
-        if (\is_callable(['parent', 'expectException'])) {
+        if (\is_callable([parent::class, 'expectException'])) {
```
Comment (keradus) [inline src/PolyfillTrait6.php:21]: can we have a PHP CS Fixer rule that is automating that, pls? :)

**Comment by keradus** [comment]
Can we add all needed 8.x to CI, so we are sure the fix is enough to claim the 8.2 support?

File: src/PolyfillTrait6.php (hunk 1 above)
Comment (jrmajor) [reply src/PolyfillTrait6.php:21]: There is no way to determine whether `['parent', 'expectException']` is supposed to be a callable or just an array.

**Comment by jrmajor** [comment]
> Can we add all needed 8.x to CI, so we are sure the fix is enough to claim the 8.2 support?

Not sure I follow, what do you mean?

**Comment by keradus** [comment]
https://github.com/PHPUnitGoodPractices/polyfill/pull/19

File: src/PolyfillTrait6.php (hunk 1 above)
Comment (keradus) [reply src/PolyfillTrait6.php:21]: but if it's passed as literal to `is_callable` etc, we can do the fix

**Comment by jrmajor** [comment]
Sorry, I've mistaken this for PHPUnitGoodPractices/Traits#55 :) Rebased onto #19.

**Comment by keradus** [comment]
no worry. thanks for the work! i will release new polyfill version now

**Pull request merged**